### PR TITLE
docs: add gitlab ci to list of supported services

### PR DESCRIPTION
**Pull request by HarshCasper**
Fixes https://github.com/localstack/docs/issues/254

### CODE REVIEW

File: content/en/ci/gitlab-ci/index.md [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+    - name: localstack/localstack
+      alias: localstack
+  before_script:
+    - LOCALSTACK_URL=http://localstack:4566
+    - pip install awscli
+  script:
+    - aws s3 mb s3://test --endpoint-url ${LOCALSTACK_URL}
+    - echo "hello world" > /tmp/hello-world
+    - aws s3 cp /tmp/hello-world s3://test/hello-world --endpoint-url ${LOCALSTACK_URL}
+    - aws s3 ls s3://test/ --endpoint-url ${LOCALSTACK_URL}
```
Comment (thrau) [inline content/en/ci/gitlab-ci/index.md:47]: any reason we're not using awslocal? seems we could simplify the example a lot

**Review by thrau — COMMENTED** [review]
LGTM! great that we're adding more CI provider examples :100:

File: content/en/ci/gitlab-ci/index.md (hunk 1 above)
Comment (HarshCasper) [reply content/en/ci/gitlab-ci/index.md:47]: @thrau I experimented using `awslocal` quite a bit but it seems to only identify the `localhost:4566` which is not accessible in this case 😞

Comment (thrau) [reply content/en/ci/gitlab-ci/index.md:47]: that seems odd :thinking: `aws s3 mb s3://test --endpoint-url http://localstack:4566` is equivalent to `awslocal s3 mb s3://test`

Comment (HarshCasper) [reply content/en/ci/gitlab-ci/index.md:47]: @thrau 😞 

image

Comment (thrau) [reply content/en/ci/gitlab-ci/index.md:47]: hm, ok odd. well then :shrug: it may be worth investigating what's going on at some point.

**Pull request merged**